cogs/mod.py

```diff
@@ -34,8 +35,9 @@ def __init__(self, bot):
             message = "{}".format(
                 " ".join(args)
             )    # to work regardless of whether the person uses apostrophes
-            channel_to_send = self.bot.get_channel(
-                self.bot.config.reception_channel_id)
+            channel_to_send = utils.get(self.bot.get_guild(
```

Comment:
this should be in another issue branch and another commit. please follow [atomic commit](https://en.wikipedia.org/wiki/Atomic_commit) guidelines.